jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java

```diff
@@ -1016,4 +1020,25 @@ public void testDocumentationForSwimlane() throws Exception {
         assertEquals("Swimlane name is wrong.", lane.getName(), "Documented Swimlane");
         assertEquals("<![CDATA[Some documentation for swimlane.]]>", lane.getDocumentation().get(0).getText());
     }
+
+    @Test
+    public void testSequenceFlowPointsInsideLane() throws Exception {
+        Bpmn2JsonUnmarshaller unmarshaller = new Bpmn2JsonUnmarshaller();
```

Comment:
Hello @tsurdilo , yesterday there was merged my commit which simplifies initialization of each testcase. Definitions can be now loaded from json in the following way:
`Definitions definitions = loader.loadProcessFromJson("sequenceFlowPointsInsideLane.json");`